### PR TITLE
grid, bgCol, title

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

**Comment by ldecicco-USGS** [comment]
Oh come on -0.7% == Fail?

**Comment by jordansread** [comment]
I can change the threshold, but should I?

File: NAMESPACE [hunk 1]
```diff
@@ -3,6 +3,7 @@
 S3method(gsplot,default)
 S3method(logged,gsplot)
 S3method(print,gsplot)
+S3method(summary,gsplot)
```
Comment (jordansread) [inline NAMESPACE:6]: :+1:

**Comment by jordansread** [comment]
played around w/ it. Looks good to me :+1:

File: R/grid.R [hunk 2]
```diff
@@ -11,15 +11,84 @@
 #' gsNew <- points(gs, y=1, x=2, xlim=c(0,NA),ylim=c(0,NA),
 #'             col="blue", pch=18, legend.name="Points")
 #' gsNew <- lines(gsNew, c(3,4,3), c(2,4,6), legend.name="Lines")
-#' gsNew <- grid(gsNew) 
+#' gsNew <- grid(gsNew, legend.name="Grid")
```
Comment (ldecicco-USGS) [inline R/grid.R]: oh yeah, I don't think the legend works...note to self: test this on computer with R....

Comment (lindsayplatt) [inline R/grid.R]: we would need to add `set_legend_args(object, fun.name=fun.name, ..., legend.name=legend.name)` to grid.gsplot for this to work

Comment (jordansread) [inline R/grid.R]: Should we include grid in the legend? I was thinking no, just like we don't include ticks in the legend

Comment (lindsayplatt) [inline R/grid.R]: I also thought no, so I didn't include it in the changes

**Comment by jordansread** [comment]
good to merge?

File: R/grid.R (hunk 2 above)
Comment (ldecicco-USGS) [inline R/grid.R]: OK, having the grid in the legend was in an example of ours a looong time ago. I also don't like the idea, so I'll quick take this out of the example, and merge the final pull request.

**Comment by jordansread** [comment]
LDC has some more things to add I think. holding off on merge

**Comment by ldecicco-USGS** [comment]
I'll merge then send to GRAN after the checks come back

**Comment by lindsaycarr** [comment]
do horizontal grid lines work with reverse axes now?

**Comment by ldecicco-USGS** [comment]
yup

**Comment by ldecicco-USGS** [comment]
horizontal worked before, just not with a reverse axis...but anyway, fixed

**Comment by lindsayplatt** [comment]
^ yeah, I hadn't read through all of the notifications before asking that!

**Pull request merged**